core/src/cljstyle/format/ns.clj

```diff
@@ -205,7 +205,7 @@
   (when (seq elements)
     (->> elements
          (mapcat expand-require-group)
-         (sort-by libspec-sort-key)
+         (sort-by (comp name libspec-sort-key))
```

Comment:
Since the purpose of `libspec-sort-key` is to produce something safe to sort across all possible libspecs, I think the string-ifying should happen directly in it.